docs/modules/ROOT/pages/contributing/general_guidelines.adoc

```diff
@@ -7,6 +7,8 @@ This document describes the general guidelines for contributing to the DevOps St
 
 The DevOps Stack is a collection of modules, each of them having its own release cycle, in order to ease the development and maintenance of each module.
 
+TIP: We have created a private GitHub project owned by the https://github.com/orgs/camptocamp/teams/is-devops-stack/[`@camptocamp/is-devops-stack` team] that is available https://github.com/orgs/camptocamp/projects/3/[here]. It is a useful way to follow the progress of the PRs and Issues of all the repositories. For more information on how it is implemented, check the xref:ROOT:contributing/project_board.adoc[Project Board] page.
```

Comment:
Generally speaking, the documentation should describe what exists, how to do things, etc. instead of a work log (we did this, we created that). That makes for less confusing reading later on when things won't be new anymore. So I suggest rephrasing like this:

```suggestion
TIP: A private GitHub project owned by the https://github.com/orgs/camptocamp/teams/is-devops-stack/[`@camptocamp/is-devops-stack` team] is available https://github.com/orgs/camptocamp/projects/3/[here]. It is a useful way to follow the progress of the PRs and Issues of all the repositories. For more information on how it is implemented, check the xref:ROOT:contributing/project_board.adoc[Project Board] page.
```

Reply:
Excellent suggestion. It's a good point what you said on the other comment about reading this 1 year from now :wink: